setup.py

```diff
@@ -74,13 +74,11 @@ def parse_requirements(filename):
         # Specify the Python versions you support here. In particular, ensure
         # that you indicate whether you support Python 2, Python 3 or both.
         'Programming Language :: Python',
-        'Programming Language :: Python :: 2',

```

Comment:
Not a biggie, but `'Programming Language :: Python :: 2',` could stay because at least one 2.x is supported.